.gitignore

```diff
@@ -121,3 +121,13 @@ ENV/
 
 # emacs buffers
 \#*
+
+runinfo*
+parsl/tests/.pytest*
+
+# documentation generation
+docs/stubs/*
+docs/1-parsl-introduction.ipynb
+
+/tmp
+parsl/data_provider/dyn.new.py
```

Comment:
I think some of these don't need to be here - like dyn.new.py
Some of these are probably worth adding: docs/stubs, docs/1.... for example - and maybe you can make a separate PR to do that because this isn't provenance related, but more to do with developer experience.

Reply:
will do